clients/banking/src/components/TransferInternationalWizardAmount.tsx

```diff
@@ -132,9 +137,43 @@ export const TransferInternationalWizardAmount = ({
     })
     .otherwise(() => []);
 
+  const metadata = match(quote)
+    .with(
+      AsyncData.P.Done(
+        Result.P.Ok({ internationalCreditTransferQuote: P.select(P.not(P.nullish)) }),
+      ),
+      quote => ({
+        rate: quote.exchangeRate,
+        total: quote.sourceAmount as Amount,
+        out: {
+          value: `${parseFloat(quote.feesAmount.value) + parseFloat(quote.sourceAmount.value)}`,
+          currency: quote.sourceAmount.currency,
+        } as Amount,
+      }),
+    )
+    .otherwise(() => undefined);
+
   return (
     <View>
-      <Tile>
+      <Tile
+        footer={
+          errors.length > 0 ? (
+            <LakeAlert
+              anchored={true}
+              variant="error"
+              title={
+                errors.length > 1 ? t("transfer.new.internationalTransfer.errors.title") : errors[0]
+              }
+            >
+              {errors.length > 1
+                ? errors?.map((message, i) => (
+                    <LakeText key={`validation-alert-${i}`}>{message}</LakeText>
+                  ))
+                : null}
+            </LakeAlert>
+          ) : null
+        }
```

Comment:
I think this conditions are a bit difficult to understand.
If you use pattern matching instead:

```tsx
<Tile
  footer={match(errors)
    .with([], () => null) // no error
    .with([P.select()], error => <LakeAlert anchored={true} variant="error" title={error} />) // 1 error
    .otherwise(errors => ( // 1+ errors
      <LakeAlert
        anchored={true}
        variant="error"
        title={t("transfer.new.internationalTransfer.errors.title")}
      >
        {errors.map((message, index) => (
          <LakeText key={`validation-alert-${index}`}>{message}</LakeText>
        ))}
      </LakeAlert>
    ))}
>
```

It's easier to read, IMHO. WDYT?